apparmor.d/groups/lxqt/featherpad

```diff
@@ -0,0 +1,57 @@
+# apparmor.d - Full set of apparmor profiles
+# Copyright (C) 2024 Alexandre Pujol <alexandre@pujol.io>
+# Copyright (C) 2024 Besanon  <m231009ts@mailfence.com>
+# SPDX-License-Identifier: GPL-2.0-only
+
+#include <tunables/global>
+
+@{exec_path} = @{bin}/featherpad
+profile featherpad @{exec_path} {
+  include <abstractions/base>
+  include <abstractions/common/app>
+  include <abstractions/gtk>
+  include <abstractions/gvfs-open>
+  include <abstractions/video>
+  include <abstractions/lxqt>
+  include <abstractions/thumbnails-cache-read>
+  include <abstractions/qt5-shader-cache>
+  include <abstractions/dbus-session-strict>
+  include <abstractions/dbus-accessibility-strict>
```

Comment:
These abstractions won't work in apparmor.d. You need to use:
```
include <abstractions/bus-accessibility>
include <abstractions/bus-session>
```

Same in any other profiles you use it